### DISCUSSION TITLE
Line highlighting for beamer

### DISCUSSION THREAD

**Discussion by jlperla** [quarto-dev/quarto-cli#6675] [Feature Requests]
### Description

It sounds like the beamer doesn't support any highlighting of line numbers?  The link https://quarto.org/docs/reference/formats/presentations/beamer.html#code  says "For Reveal Only" but this is in the beamer section, so I wanted to verify I wasn't missing anything?

Basically, I want to be able to show things like
    
    ```{.python code-line-numbers="6-8"}
    import numpy as np
    import matplotlib.pyplot as plt
    
    r = np.arange(0, 2, 0.01)
    theta = 2 * np.pi * r
    fig, ax = plt.subplots(subplot_kw={'projection': 'polar'})
    ax.plot(theta, r)
    ax.set_rticks([0.5, 1, 1.5, 2])
    ax.grid(True)
    plt.show()
    ```

From the manual.  I do know that both the latex `minted` and `lstlisting` support highlighting lines in some way.  Is there a hack to use them?

**Comment by cscheid** [comment]
> but this is in the beamer section, so I wanted to verify I wasn't missing anything?

No, unfortunately you're not. Our documentation infrastructure doesn't currently have the ability to produce different documentation strings for different formats when both formats accept the same key.

> I do know that both the latex `minted `and `lstlisting` support highlighting lines in some way. Is there a hack to use them?

Not that I'm aware of.
